### PR TITLE
Modified all sight-values

### PR DESCRIPTION
**Pull request by Micr0Bit**
Modified all sight-values for units and buildings to match the original rules

Part of #17

### CODE REVIEW

File: rules/concrete.yaml [hunk 1]
```diff
@@ -4,7 +4,9 @@
 		TerrainTypes: Rock
 		BuildSounds: CHUNG.WAV
 		AllowInvalidPlacement: true
-		Adjacent: 1
+    		Adjacent: 1
```
Comment (MustaphaTR) [inline rules/concrete.yaml]: Looks like you changed the tabs to spaces here.

**Comment by MustaphaTR** [comment]
IIRC, original had different sight values for units when moving and when stopped. I'm not sure it is currently possible to replicate that on OpenRA tho.

**Comment by Micr0Bit** [comment]
@MustaphaTR : not sure where you got the information about different sight-values during movement ...

**Comment by MustaphaTR** [comment]
It is like that in the Android remake i play(which is complate remake, no difference from original afaik). I never played it on original game on DOS so i can't say anything clear. If you want i can send a screen record.

**Comment by Micr0Bit** [comment]
@MustaphaTR : im not using a "android remake" as a reference to justify my changes ... but rather the actually dos-game that you can download on several abandoned-games sites ... and run via dosbox

**Comment by MustaphaTR** [comment]
I"ll take a look, i had the DOS version on Vitural Box on PC but i don't remember if i lost it with a format or not. Can't do it now, possiby tomorrow.

**Comment by Micr0Bit** [comment]
siegediff

i see what you mean , you are right ... that is very interesting 

i have to check all the units

**Comment by MustaphaTR** [comment]
Needs rebase.

I think this can go after someone else confirm those are the correct values, i don't have time for it now. We can solve #40 on another PR.

**Pull request closed without merging**